scripts/partition_manager.py

```diff
@@ -826,7 +826,7 @@ def verify_static_conf_simple(size, start, placement_strategy, static_conf):
         return
 
     if placement_strategy == START_TO_END:
-        start_end_correct = gaps[0][0] == start + size
+        start_end_correct = gaps[0][1] == start + size
```

Comment:
Could you explain this change a bit more?

Reply:
I was struggling to understand the code that produces `gaps`, but the result is a list of tuples with the start address in `0` and end address in `1`. Before we compared the start of the first gap to be equal to the end of the region (start + size). The fix is to instead compare the end of the first gap to the end of the region.

Reply:
Thank you :)